proposal.md

```diff
@@ -111,10 +111,18 @@ SEE ALSO (map to relevant subsections):
 #### Help pages
 
 Source files for help pages use the "R documentation" (Rd) format. 
-`utils::prompt` and `utils::promptData` may be used to create an Rd template for a function or data set, respectively. `tools::Rdcheck` may be used to validate Rd files, e.g., detecting syntax errors.
+`utils::prompt()` and `utils::promptData()` may be used to create an Rd template for a function or data set, respectively. `tools::checkRd()` may be used to validate Rd files, e.g., detecting syntax errors.
 
 WRE reference: [Writing R documentation files](https://cran.r-project.org/doc/manuals/r-release/R-exts.html#Writing-R-documentation-files)
 
+- `r pkg("roxygen2", priority = "core")` provides the `roxygenise()` function to generate Rd files from roxygen blocks: comments with specific markup in R source files. When used with Markdown, roxygen can greatly reduce the markup required in the documentation source. Several [development workflow](#development-workflow) packages support creating documentation with roxygen. `r pkg("Rdroxygen")` provides functions to convert Rd files to roxygen blocks; `r pkg("roxygen2md")` replaces Rd syntax in roxygen comments with the Markdown equivalent.
```

Comment:
Several suggestions:

```suggestion
- `r pkg("roxygen2", priority = "core")` provides the `roxygenise()` function to generate Rd files from comments with specific markup in R source files. When used with Markdown, roxygen2 can greatly reduce the markup required in the documentation source. Several [development workflow](#development-workflow) packages support creating documentation with roxygen2. 
     `r pkg("Rd2roxygen")` provides functions to convert Rd files to roxygen2 comments.
     `r pkg("roxygen2md")` replaces Rd syntax in roxygen2 comments with the Markdown equivalent.
```

Perhaps not using "roxygen blocks" is easier for people not familiar with the package. 
I used roxygen2 to be consistent with the package name. 
I also separated the two packages that manipulate roxygen2 comments into a subparagraph (not sure how it renders as I'm editing this via github).

Reply:
Yes, I was also unsure about this. If we're go with using the package name consistently as you suggest, it's probably best to always mark it up as a package name. So I'll accept this suggestion and update the formatting, checking the sub paragraphs at the same time.